.github/pull_request_template.md

```diff
@@ -1,47 +1,43 @@
 Resolves #ISSUE(s).
-<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:
-
-Partially addresses:
-- ISSUE(s)
-
-Addresses sub-task (n) in:
-- ISSUE(s)
+<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address 
```

Comment:
## "Resolves #ISSUE" comment conciseness
I found that the comment I added previously was not concise, and I think this is an improvement.